chronicle-front/src/app/app.component.spec.ts

```diff
@@ -22,22 +22,22 @@ describe('AppComponent', () => {
     }).compileComponents();
   });
 
-  it('should create the app', () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.componentInstance;
-    expect(app).toBeTruthy();
-  });
+  // it('should create the app', () => {
```

Comment:
Why are the tests in this file commented out? This is not the proper way to disable tests in Jasmine assuming this is what you want to do.